### PR TITLE
Spaceranger Logical Turtle Post

### PR DESCRIPTION
**Pull request by spaceranger-23**

### CODE REVIEW

File: _posts/spaceranger-23/2020-09-09-logical-turtle-post-spaceranger-23.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+author: spaceranger-23
+title: "spaceranger Logical Turtle Post"
+---
+
+Attach code
+
```
Comment (cchinchia) [inline _posts/spaceranger-23/2020-09-09-logical-turtle-post-spaceranger-23.md:8]: let me know when you attach embed code

**Review by cchinchia — CHANGES_REQUESTED** [review]
Almost there!

File: _posts/spaceranger-23/2020-09-09-logical-turtle-post-spaceranger-23.md (hunk 1 above)
Comment (spaceranger-23) [reply _posts/spaceranger-23/2020-09-09-logical-turtle-post-spaceranger-23.md:8]: Hey, sorry for the delay. Just attached to the code.

**Review by spaceranger-23 — COMMENTED** [review]
Updated Logical Turtle

**Review by cchinchia — APPROVED** [review]
👍

**Pull request merged**